test/education.e2e-spec.ts

```diff
@@ -30,15 +30,15 @@ describe('Education e2e', () => {
   });
 
   it('create a education', async () => {
-    const education = await createEducation(app, { userId: user.id });
-    expect(education.id).toBeDefined();
+    const education = await createEducation(app, { userId: user?.id });
+    expect(education?.id).toBeDefined();
   });
 
   it('read one education by id', async () => {
-    const education = await createEducation(app, { userId: user.id });
+    const education = await createEducation(app, { userId: user?.id });
 
     try {
-      const { education: actual } = await app.graphql.query(
+      const result = await app.graphql.query(
```

Comment:
`result` should not be nullable...

Reply:
I mean there shouldn't be a need for the optional chaining on this variable

Reply:
The real issue here was in `create-graphql-client.ts`, where the query is typed to return a Promise resolving to potentially `null` or `undefined` (see snippet below). 

Thoughts on changing this to always assume a payload is returned?

```
export interface GraphQLTestClient {
  query: (
    query: DocumentNode,
    variables?: { [name: string]: any },
  ) => Promise<Record<string, any> | null | undefined>;
  mutate: (
    mutation: DocumentNode,
    variables?: { [name: string]: any },
  ) => Promise<Record<string, any> | null | undefined>;
  authToken: string;
}
```

Reply:
Ah. Ok yeah let's remove the nullable output from that interface. The `return` statements in the implementation will just need a non-null assertion.

Reply:
Sounds good, will do